upgrade-tidb-using-tiup.md

```diff
@@ -169,11 +170,13 @@ tiup cluster upgrade <cluster-name> v5.3.0
 
 > **Note:**
 >
-> + Performing a rolling upgrade to the cluster will upgrade all components one by one. During the upgrade of TiKV, all leaders in a TiKV instance are evicted before stopping the instance. The default timeout time is 5 minutes (300 seconds). The instance is directly stopped after this timeout time.
+> + An online upgrade upgrades all components one by one. During the upgrade of TiKV, all leaders in a TiKV instance are evicted before stopping the instance. The default timeout time is 5 minutes (300 seconds). The instance is directly stopped after this timeout time.
 >
 > + To perform the upgrade immediately without evicting the leader, specify `--force` in the command above. This method causes performance jitter but not data loss.
 >
 > + To keep a stable performance, make sure that all leaders in a TiKV instance are evicted before stopping the instance. You can set `--transfer-timeout` to a larger value, for example, `--transfer-timeout 3600` (unit: second).
+>
+> - You cannot upgrade TiFlash online from versions earlier than v5.3 to v5.3 or later. To achieve such a cross-version upgrade, you need to stop the TiFlash instances first and upgrade the cluster offline. Then, reload the cluster so that other components are upgraded online without interruption.
```

Comment:
```suggestion
> - You cannot upgrade TiFlash online from versions earlier than v5.3 to v5.3 or later. Instead, you can stop the TiFlash instances first and upgrade the cluster offline. Then, reload the cluster so that other components are upgraded online without interruption.
```